src/Console/FieldGeneratorCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Why do we need this abstract class instead of implement it in QueryCommand?

Reply:
Good catch. Generating mutations uses the same stub and should work the same, i forget to change the command class. 5a0f1137ac144a69c5dda6c28d4255ad9bbd5157